adding-behavior/changing-the-dom.md

```diff
@@ -0,0 +1,255 @@
+# Changing the DOM
+
+## Goals
+
+The goal for this lesson is to display one of JavaScript's strengths: accessing and changing the DOM.
+
+The skills of changing the DOM using vanilla JavaScript will transfer to other front-end skills, such as working with React.
+
+## Introduction
+
+Keira is a crab enthusiasm, and her enthusiasm for crustaceans goes as deep as the ocean. She's making a fan site for crabs. She's already made a draft website for it.
+
+However, she's in a very strange situation-- she knows what her `index.html` file will look like when it loads... and she wants to change how it will appear _after_ it loads. Keira needs to change the content and appearance of her site _after_ it loads in the browser, _without_ changing the HTML and CSS.
+
+How can Keira accomplish her goal? She'll use JavaScript and her knowledge of the DOM to help her!
+
+To change elements in the DOM, we'll need to learn two things:
+
+- How to select HTML elements in the DOM
+- How to specify the changes we want
+
+## HTML Data in `document`
+
+To work with the DOM, we need to learn how to access the DOM. To do that, let's run an experiment:
+
+1. With _any_ website open, open the browser Dev Tools
+1. Open the console
+1. Run the statement `document`
+
+We should see the details of a document object print to the console.
```

Comment:
since we're referring to the document just above, the definite article is more appropriate
```suggestion
We should see the details of the document object print to the console.
```